.gitignore

```diff
@@ -1,160 +1,19 @@
-bower_components/
-node_modules/
-bower_cmps/
-
-## Ignore Visual Studio temporary files, build results, and
-## files generated by popular Visual Studio add-ons.
-
 # User-specific files
-*.suo
-*.user
-*.sln.docstates
-*.sln.ide/
-applicationhost.config
-v15/
 .nuget/
+.vs/
 
-# NServiceBus Logs
-nsb_log_*.txt
-
-# Build results
-
-[Dd]ebug/
-[Rr]elease/
-x64/
-build/
+# Build artifacts
 [Bb]in/
 [Oo]bj/
 
-# Enable "build/" folder in the NuGet Packages folder since NuGet packages use it for MSBuild targets
-!packages/*/build/
-
-# MSTest test Results
-[Tt]est[Rr]esult*/
-[Bb]uild[Ll]og.*
-
-*_i.c
-*_p.c
-*.ilk
-*.meta
-*.obj
-*.pch
-*.pdb
-*.pgc
-*.pgd
-*.rsp
-*.sbr
-*.tlb
-*.tli
-*.tlh
-*.tmp
-*.tmp_proj
-*.log
-*.vspscc
-*.vssscc
-.builds
-*.pidb
-*.log
-*.scc
-
-# Visual C++ cache files
-ipch/
-*.aps
-*.ncb
-*.opensdf
-*.sdf
-*.cachefile
-
-# Visual Studio profiler
-*.psess
-*.vsp
-*.vspx
-
-# Guidance Automation Toolkit
-*.gpState
+# Nuget packages
+packages/
 
 # ReSharper is a .NET coding add-in
+
```

Comment:
Why has this line been added?

Reply:
It happened to me that sometimes R# detritus are included. I’ve not been using R# for a long time now, but there are people in the company that do use it. Are you using it @adamralph ?

Reply:
Sorry, I didn't make myself clear. I just meant the extra blank line after the comment. The other sections don't have it.

No, I don't use R#. 😉

Reply:
A typo

Reply:
Fixed